### DISCUSSION TITLE
`error: failed to fetch` when running `scarb build`

### DISCUSSION THREAD

**Discussion by ghost** [OpenZeppelin/cairo-contracts#718] [Q&A] [answered]
I'm following along with the `README.md` here (https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/README.md) to implement the ERC721. But when I run `scarb build`, I receive this error

```
Updating git repository https://github.com/openzeppelin/cairo-contracts
error: failed to fetch

Caused by:
    process did not exit successfully: exit status: 128
```

Any idea on this error?

**Comment by ericnordelo** [comment]
Which scarb version are you using? Make sure is:

```
$ scarb --version

scarb 0.7.0 (58cc88efb 2023-08-23)
cairo: 2.2.0 (https://crates.io/crates/cairo-lang-compiler/2.2.0)
sierra: 1.3.0
```

**Reply by ghost** [reply, marked as answer]
I managed to solve the problem, the OZ's README.md has a typo, the tag is currently "v0.7.0-rc.0"
